src/Context/Filter_context.js

```diff
@@ -0,0 +1,23 @@
+import { createContext, useContext, useReducer } from "react";
+import { filterReducerFunc } from "../reducers/FilterReducer";
+
+const FilterContext = createContext()
+
+const useFilter = () => useContext(FilterContext)
+
+const FilterProvider = ({children}) => {
+
+    const [state,dispatch] = useReducer(filterReducerFunc,{
```

Comment:
Suggestion: you can change the variable name of state, dispatch for better code readability when you add more Context Provider

Reply:
Change Variable name of state

Reply:
But it might unmark the coding pattern